package.json

```diff
@@ -23,11 +23,14 @@
     "test": "mocha tests/ --recursive"
   },
   "dependencies": {
+    "eslint-plugin-babel": "4.1.2",
```

Comment:
Do we want only some of these versions pinned but not others?

Reply:
Not really sure. I suppose we can pin them all, yeah?

Reply:
Sounds like a plan :)